services/QuillLMS/engines/evidence/app/controllers/evidence/research/gen_ai/dataset_relevant_texts_controller.rb

```diff
@@ -4,6 +4,10 @@ module Evidence
   module Research
     module GenAI
       class DatasetRelevantTextsController < ApplicationController
+        def index
+          render json: { dataset_relevant_texts: dataset.dataset_relevant_texts.order(:id) }
```

Comment:
I like the explicit ordering by `id` here.  I think it's smart not to leave that to chance.